spec/02-integration/05-proxy/15-upstream-status-header_spec.lua

```diff
@@ -130,13 +130,17 @@ describe(constants.HEADERS.UPSTREAM_STATUS .. " header", function()
     end)
   end)
 
-  describe("is injected with configuration [headers=X-Kong-Upstream-Status]", function()
+  for _, buffered in ipairs{false, true} do
+  describe("is injected with configuration [headers=X-Kong-Upstream-Status]" ..
```

Comment:
I would expect fix PR's to fail when the code change that added the test is removed but that's not the case here.

Sometimes this is fine I'm just clarifying that this is one of those times.